tests/expected/pointer-overflow/expected

```diff
@@ -1,2 +1,2 @@
 Status: FAILURE\
-arithmetic overflow on signed multiplication
+offset: attempt to compute number in bytes which would overflow
```

Comment:
What does "number in bytes" mean?

Reply:
Hah, good question. That's the assertion message that's there though, not part of this change.

(AFAICT, the change to this expected file is that it was previously expecting the "wrong" assertion which is no longer reached now that the "right" one stops control flow)